### DISCUSSION TITLE
配置完成后搜索资源全部返回空

### DISCUSSION THREAD

**Discussion by carolcoral** [jxxghp/MoviePilot#1080] [问题咨询]
已经完成MP的全部配置，可以正常访问TMDB、推荐等内容，通过了站点认证。
但是搜索资源返回“0”，通过查看后台日志，抓取服务发送的请求进行浏览器直接访问又是可以正常搜索到资源内容的。

请问这种情况需要怎么处理？或者说我的MP哪里设置不对？

![image](https://github.com/jxxghp/MoviePilot/assets/8952687/41c8b85f-17bc-4638-9bd3-f823e7b28420)
![image](https://github.com/jxxghp/MoviePilot/assets/8952687/fc3538a1-dd0d-41d3-afeb-8814f4c13f0c)

![image](https://github.com/jxxghp/MoviePilot/assets/8952687/aa39d6d5-ac45-4344-8451-4f1a8581d8dd)
![image](https://github.com/jxxghp/MoviePilot/assets/8952687/d17d0b4f-b535-4986-bec1-c768c893aa3b)

### 我的相关设定：

![image](https://github.com/jxxghp/MoviePilot/assets/8952687/a4150ac0-d041-4d42-a465-d7a7b9467528)

* 搜索中的`默认过滤规则`未填写使用的默认值
![image](https://github.com/jxxghp/MoviePilot/assets/8952687/b53631eb-7658-4a21-8985-e90581e4721a)

![image](https://github.com/jxxghp/MoviePilot/assets/8952687/2bc445db-ace5-4cee-bb75-9bcf552d2d9c)

**Comment by carolcoral** [comment]
#### 补充内容：
<img width="986" alt="image" src="https://github.com/jxxghp/MoviePilot/assets/8952687/f7cd0a48-c3f9-4b1b-af65-51c4ebe7d379">

**Comment by carolcoral** [comment]
> 前提：使用了相同的RSS地址

* 用NASTOOL可以搜索到资源
<img width="1222" alt="image" src="https://github.com/jxxghp/MoviePilot/assets/8952687/40da9263-31ce-4761-832a-a02c2e8fff11">

* 使用MoviePilot无法搜索到资源
<img width="1150" alt="image" src="https://github.com/jxxghp/MoviePilot/assets/8952687/dad8d4f3-b9c9-4d6f-a2cf-514e73d39162">

**Comment by carolcoral** [comment]
有人能解决么，升级到1.4.3还是这样

**Comment by codeses** [comment]
我也遇到这问题 你解决了吗

**Reply by carolcoral** [reply]
没有

**Comment by zzzh2023** [comment]
一年了，解决了吗，我刚装完也是这个情况，一模一样。

**Reply by carolcoral** [reply]
解决了，你要保证你能访问到TMDB等资源才行

**Comment by qiniuweihe** [comment]
怎么保证，openclash 添加规则吗？

**Reply by carolcoral** [reply]
> 怎么保证，openclash 添加规则吗？

不用，我的方案是局域网挂了一个clash，mp直接走内部网络访问就行，而且mp也支持手动改host，自己找到可用的ip改下hosts就行

**Comment by gzxiexl** [comment]
见鬼了 我升级到最新的docker后重新配置 跟你的情况一样 
![image](https://github.com/user-attachments/assets/2894f5fc-c4a8-4d6f-b5fc-0a748a7fefa8)
网络都没问题，请求的链接 浏览器中访问都有资源 但是订阅搜索就是无资源

**Reply by carolcoral** [reply]
是不是站点cookie的问题

**Reply by gzxiexl** [reply]
站点测试正常，然后我不用MP自带的cloudcookie服务端，额外自建了一个，又可以了
